internal/config/analytics.go

```diff
@@ -17,8 +18,18 @@ type AnalyticsConfig struct {
 
 // ClickhouseConfig defines the connection details for connecting Flipt to Clickhouse.
 type ClickhouseConfig struct {
-	Enabled bool   `json:"enabled,omitempty" mapstructure:"enabled" yaml:"enabled,omitempty"`
-	URL     string `json:"url,omitempty" mapstructure:"url" yaml:"url,omitempty"`
+	Enabled  bool   `json:"enabled,omitempty" mapstructure:"enabled" yaml:"enabled,omitempty"`
+	URL      string `json:"url,omitempty" mapstructure:"url" yaml:"url,omitempty"`
+	Username string `json:"username,omitempty" mapstructure:"username" yaml:"username,omitempty"`
+	Password string `json:"password,omitempty" mapstructure:"password" yaml:"password,omitempty"`
```

Comment:
```suggestion
	Password string `json:"-" mapstructure:"password" yaml:"password,omitempty"`
```

we don't want to expose these publicly over the `/meta/config` endpoint